### PR TITLE
Fixing product-iots#1322

### PR DESCRIPTION
**Pull request by dilee**
Fixed https://github.com/wso2/product-iots/issues/1322

### CODE REVIEW

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js [hunk 1]
```diff
@@ -46,6 +46,22 @@ $(document).ready(function() {
 
 });
 
+function getLogStatusIcon(entry) {
+    switch (entry) {
```
Comment (harshanL) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js:50]: You don't need to have break after return as returning the function will terminate the flow.

Comment (dilee) [reply components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js:50]: Refactored and fixed

**Review by harshanL — APPROVED** [review]

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js [hunk 2]
```diff
@@ -46,6 +46,19 @@ $(document).ready(function() {
...
+        case 'COMPLETED':
+            return 'fw-success';
+        case 'PENDING':
+            return 'fw-pending';
+        case 'ERROR':
+            return 'fw-error';
+        default:
+            return 'fw-info'
```
Comment (charithag) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js:58]: Add semicolon by terminating the statement

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js [hunk 3]
```diff
@@ -166,11 +179,11 @@ function loadOperationsLog(update) {
...
-                    break;
+                    return 'fw-success';
                 case 'PENDING':
-                    return 'fw-pending'
-                    break;
+                    return 'fw-pending';
+                case 'ERROR':
+                    return 'fw-error';
                 default:
                     return 'fw-info'
```
Comment (charithag) [inline components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js:188]: Add semicolon by terminating the statement

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js (hunk 2 above)
Comment (dilee) [reply components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js:58]: Fixed in https://github.com/wso2/carbon-device-mgt/pull/950

File: components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js (hunk 3 above)
Comment (dilee) [reply components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js:188]: Fixed in https://github.com/wso2/carbon-device-mgt/pull/950

**Pull request merged**